src/shared/components/enhancedReactTable/EnhancedReactTable.tsx

```diff
@@ -148,12 +176,24 @@ export default class EnhancedReactTable extends React.Component<IEnhancedReactTa
                     handleInput={this.handleFilterInput}
                     onColumnToggled={this.handleVisibilityToggle}
                     showSearch={true}
+                    className="pull-right"
+                    paginationProps={{itemsPerPage:this.state.itemsPerPage,
```

Comment:
Should we create a secondary `TableHeaderControls` at the end of the table for pagination?